torch/csrc/jit/pickler.h

```diff
@@ -85,7 +85,13 @@ enum class OpCode : char {
   FRAME = '\x95'
 };
 
-enum PicklerClass : uint8_t { TENSOR = 0, INTLIST = 1 };
+enum PicklerClass : uint8_t {
+  TENSOR = 0,
+  INTLIST = 1,
+  TENSORLIST = 2,
```

Comment:
Do we need to increase the Op version now?

Reply:
Adding these new enum values should have any BC effect

Reply:
Is op version just for marking backwards incompatibility? I thought it was also for detecting if you forward-incompatibility mismatches

Reply:
I don't think we've made (or should make) any guarantees about forward compatibility

Reply:
It isn't a guarantee about forward compatibility; it's for easy detection that there is a version mismatch. cc: @dzhulgakov

Reply:
If you add new functionality, it helps to bump the version number. The error is just "hey you are trying to load a new model with an old version of pytorch" but it is an important error to report. Teams need to know when they have to rebuild and deploy their services.